app/code/Magento/Search/composer.json

```diff
@@ -11,7 +11,8 @@
         "magento/module-catalog-search": "*",
         "magento/module-reports": "*",
         "magento/module-store": "*",
-        "magento/module-ui": "*"
+        "magento/module-ui": "*",
+        "magento/module-advanced-search": "*"
```

Comment:
I think that Seach should not depend on Advanced search, the dependency should be reversed

Reply:
Hi @sivaschenko, thanks for your review. I was wondering how to access the constant in AdvancedSearch without adding a dependency. :thinking: All the functionality is present in Search module but only the string constant is present in AdvanceSearch for some reason.

Reply:
If the AdvancedSearch functionality is causing this issue, it should be fixed in the AdvancedSearch module. Probably, in this case, it requires rewriting a template.